pkg/tf2pulumi/convert/tf.go

```diff
@@ -1423,6 +1423,43 @@ func convertBody(sources map[string][]byte, scopes *scopes, fullyQualifiedPath s
 	return newAttributes
 }
 
+// camelCaseObjectAttributes rewrites the attributes of objects to camelCase and returns the modified value.
+// when the input is a list of objects or map of objects, those are modified recursively.
+func camelCaseObjectAttributes(value cty.Value, valueType cty.Type) cty.Value {
```

Comment:
Do we really need the type here? The cty.Value knows it's type, no need to pass in another variable (that might be wrong if we mess up the call to this)

Reply:
I used this because `config.Type` might not be the same as `config.Default.Type()` but I guess we should go off of the `config.Default : cty.Value` in this case. I've simplified the implementation now.